src/Illuminate/Database/Eloquent/FactoryBuilder.php

```diff
@@ -344,10 +344,7 @@ protected function stateAttributes($state, array $attributes)
             return $stateAttributes;
         }
 
-        return call_user_func(
-            $stateAttributes,
-            $this->faker, $attributes
-        );
+        return $stateAttributes($this->faker, $attributes);
```

Comment:
Here it's checking for `is_callable()`, if you give `[$object, 'bar']` it going to return `true`, but AFAIK it not possible to do `[$object, 'bar'](...)`

Reply:
Yeh, you can call callables like that. The two cases I didn't convert are ones with assignments to the variable holding the callable within the call_user_func parameters, and callables which were properties of classes.